filter_plugins/config_encoders.py

```diff
@@ -520,7 +520,7 @@ def encode_json(
 
 def encode_logstash(
         data, convert_bools=False, convert_nums=False, indent="  ", level=0,
-        prevtype="", section_prefix=":", backslash_escaping_ignore_string='@@@'):
+        prevtype="", section_prefix=":", backslash_ignore_prefix='@@@'):
```

Comment:
Please could you reflect the parameter change also in the README?

Reply:
The README should be already reflecting the changes in the code.